mtags/src/main/scala/scala/meta/internal/mtags/Semanticdbs.scala

```diff
@@ -123,3 +126,22 @@ object Semanticdbs {
       nonDefaultRelPath: Option[RelativePath]
   )
 }
+
+object Shebang {
+  private val sheBangRegex: Regex = s"""(^(#!.*(\\r\\n?|\\n)?)+(\\s*!#.*)?)""".r
```

Comment:
We usually just check if the first line starts with `#!`, wouldn't that be enough?

Reply:
The issue is that md5 in semanticdb and calculated from the text need to match, so we have to do it exactly as it's done in the compiler, that is substitute all chars for spaces.

Reply:
Ach! Let's add a comment then, good catch